pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -720,6 +720,15 @@ public static class ResponseType {
       }
     }
 
+    public static class OptimizationSwitches {
```

Comment:
Suggest moving it into the `QueryOptionKey` subclass

Reply:
Moved